### ISSUE TITLE
Documentation related to changing the default ACL

### ISSUE THREAD

**Issue by yohanb** [EventStore/documentation#336]
**Describe the bug**
The examples which describe how to change the default ACL doesn't seem to work.

https://eventstore.org/docs/server/users-and-access-control-lists/index.html?tabs=tabid-http%2Ctabid-1%2Ctabid-3%2Ctabid-5#example-1

Using version 5.0.0

**To Reproduce**
1. Create a `reader` and a `writer` user with no additional permissions
2. Go to https://eventstore.org/docs/server/users-and-access-control-lists/index.html?tabs=tabid-http%2Ctabid-1%2Ctabid-3%2Ctabid-5#example-1
3. Reproduce the curl example with the following user ACL:
```
...
	"$userStreamAcl": {
		"$w": "writer",
		"$d": "writer",
		"$mw": "writer"
	}
...
```
4. Create a new stream with the `writer` user
5. Create a new stream with the `reader` user (works but shouldn't)

**Expected behavior**
The `reader` user gets a 401 unauthorized when trying to create a stream

**Additional context**
Was able to make it work but not like the documentation describes it. It worked with the following `override-default.json`:
```
	"$userStreamAcl": {
		"$w": "writer",
		"$d": "writer",
		"$mw": "writer"
	}
```

`curl -i -d @override-default.json http://127.0.0.1:2113/streams/%24settings --user admin:xxx -H "Content-Type: application/json" -H "ES-EventId: xxx" -H "ES-EventType: update-default-acl"`

So basically, when writing event data to the `$settings` stream, not metadata to the `$settings/metadata` stream.

**Comment by ChrisChinchilla** [comment]
Thanks @yohanb I thought I had checked this, but maybe something changed.
